MaintenanceManager/CHANGELOG.md

```diff
@@ -16,10 +16,14 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
-## [1.0.16] - 2023-04-12
+## [1.0.17] - 2023-04-12
 ### Fixed
 - Fixed IARM_Bus_UnRegisterEventHandler  call to IARM_Bus_RemoveEventHandler
 
+## [1.0.16] - 2023-04-11
```

Comment:
this should come as 17 we should always use the new number while mering to main branch